docs/source/release_notes/release-notes.rst

```diff
@@ -45,6 +45,12 @@ v0.22.0
          - The ``clean-cass-locks-state`` CLI now clears the lock instead of dropping the locks table.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1056>`__)
 
+    *    - |new|
+         - Added the ability to do custom performance logging for Sweep. This is currently only available for
```

Comment:
Scratch that, on second thought we can probably just tell people to temporarily bump the log level of `com.palantir.atlasdb.sweep` to DEBUG.

Would other internal products ever want to have special snowflake perf logging?